components/src/lib/types/Button.type.ts

```
@@ -0,0 +1,6 @@
+export interface ButtonProps {
+  onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void | (() => void);
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)